pkg/zero/api.go

```diff
@@ -0,0 +1,123 @@
+// Package zerosdk contains the pomerium zero configuration API client
+package zerosdk
```

Comment:
Should this match the directory name? i.e. either move this to `pkg/zerosdk` or change this to `package zero`?